command_line_assistant/config.py

```diff
@@ -33,9 +33,36 @@
 [backend]
 endpoint = "{endpoint}"
 verify_ssl = {verify_ssl}
+
+[logging]
+type = "{logging_type}"
 """
 
 
+class LoggingSchema(namedtuple("Logging", ["type", "file"])):
```

Comment:
This could be a dataclass for improved readability. It's more important to be consistent with the other schema classes at this point, but it would be nice to refactor these away from a `namedtuple` at some point.

```python
import dataclasses
import typing as t

@dataclasses.dataclass(slots=True)
class LoggingSchema:
    """This class represents the [logging] section of our config.toml file."""

    _logging_types: t.ClassVar[tuple[str, str]] =  ("verbose", "minimal")
    type: str = "minimal"
    file: str | Path = "~/.cache/command-line-assistant/command-line-assistant.log"

    def _validate_logging_type(self, type: str):
        if type not in self._logging_types:
            raise TypeError(
                f"Logging type {type} is not available. Please, choose from {(',').join(self._logging_types)}"
            )

    def __post_init__(self):
        self.file = Path(self.file).expanduser()
```

Reply:
With the drop of RHEL 8 on #25, I will open a ticket to transform the namedtuple classes in dataclass

Reply:
resolving this for now